lib/Canvas.js

```diff
@@ -6,10 +6,27 @@ const CanvasApi = require('@kth/canvas-api')
 const csv = require('fast-csv')
 const tempy = require('tempy')
 const fs = require('fs')
+const Joi = require('@hapi/joi')
 
 const canvasApi = CanvasApi(process.env.CANVAS_API_URL, process.env.CANVAS_API_TOKEN)
 
+const enrollmentSchema = Joi.object().keys({
+  user_id: Joi.required(),
+  section_id: Joi.required(),
+  status: Joi.required(),
+  role_id: Joi.required()
+})
+
+/** Normalize the enrollments array to fit the schema */
+function normalizeEnrollments (enrollments) {
+  return enrollments
```

Comment:
Maybe I'm overcomplicating things, but wouldn't it be more elegant using a `reduce` for this operation? E.g.

```javascript
return enrollments.reduce((accumulator, currentValue) => {
    const result = enrollmentSchema.validate(currentValue, { stripUnknown: true })
    if (result.error === null) {
        accumulator.push(result.value)
    }
}, [])
```